vissl/engines/benchmark_suite_scheduler.py

```diff
@@ -0,0 +1,545 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
```

Comment:
question: is this scheduler supposed to run on "every gpu" ?

typically in `vissl/engines/`, we have included engines that run on every gpu. the `distributed_launcher` takes care of launching the engine on each gpu worker. 

If this doesn't run on all gpus, then move this to either `vissl/utils/` folder of within the `tools/` merge this with the new .py added as this can also facilitate better code readability (avoid many hops) :)